src/components/project/Project.tsx

```diff
@@ -103,42 +104,31 @@ export class Project extends Component<ProjectProps, ProjectState>  {
             <div>
                 <form className="white-card p-24 mb-16 dashed" onSubmit={this.saveProjectData}>
                     <div className="white-card__header"> {this.props.id ? 'Edit project' : 'Add Project'} </div>
-                    <label className="form__row">
-                        <span className="form__label dc__required-field">Project name</span>
-                        <input
-                            type="text"
-                            autoComplete="off"
-                            name="name"
-                            value={this.props.name}
-                            placeholder="e.g. My Project"
-                            className="form__input"
-                            autoFocus
```

Comment:
is autofocus removal intentional?

Reply:
Yes as generalise custom input is taking caring of autofocus, it is redundant here